### PR TITLE
Added test for acal view and all required files

### PR DESCRIPTION
**Pull request by kgirvan**

### CODE REVIEW

File: lib/ks/acal/acal.rb [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+    enrollment_home
+    search_for_calendar
+    view_acal
+    
+
+  end
+  def enrollment_home(opts={})
+
+    @request.add("/portal.do?channelTitle=Enrollment%20Home&channelUrl=#{@request.url}/kr-rad/launch?viewId=enrollmentHomeView&methodToCall=start")
+    @request.add('/kr-krad/launch?viewId=enrollmentHomeView&methodToCall=start')
```
Comment (kcampos) [inline lib/ks/acal/acal.rb:67]: There shouldn't be any hard references to the web context, "kr-rad", that's stored in the config object so you should retrieve it from there. Also if the request begins with the web context then you can ignore it all together, the core API will insert it.

File: lib/ks/acal/acal.rb [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+    puts "viewing academic calendar as: #{opts[:view_person]}"
+    puts "test name #{testname}"
+
+    enrollment_home
+    search_for_calendar
+    view_acal
+    
+
+  end
+  def enrollment_home(opts={})
```
Comment (kcampos) [inline lib/ks/acal/acal.rb:64]: Why not use the homepage method? Seems like your duplicating here

File: lib/ks/acal/acal.rb [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+
+    opts[:file_fullpath] = "#{opts[:data_dir]}/#{opts[:directory]}/#{opts[:type]}/#{opts[:filename]}"
+    
+    boundary, content = read_file(opts[:file_fullpath])
+
+    @request.add('/kr-krad/calendarSearch',
+      {
+        'method' => 'POST',
+        'content_type' => "multipart/form-data; boundary=#{boundary}",
+        #'contents' => content
```
Comment (kcampos) [inline lib/ks/acal/acal.rb:91]: Remove commented lines.

File: lib/ks/acal/acal.rb [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+            
+  end
+  
+  
+  # View an Academic Calendar
+  # TODO:
+  #
+  # Option: DEFAULT_VALUE
+  # * 'mode': 'blank'
+  def acal_view(testname, opts={})
```
Comment (kcampos) [inline lib/ks/acal/acal.rb:30]: I don't see test me variable used anywhere in the method, except for a debug statement.

File: lib/ks/acal/acal.rb (hunk 2 above)
Comment (kcampos) [inline lib/ks/acal/acal.rb:64]: Add blank link after previous method end

File: lib/ks/acal/acal.rb [hunk 5]
```diff
@@ -0,0 +1,183 @@
...
+          'Accept-Language' => 'en-us,en;q=0.5',
+          'Accept-Encoding' => 'gzip, deflate',
+          'Accept-Charset' => 'ISO-8859-1,utf-8;q=0.7,*;q=0.7',
+          'Referer' => "#{@request.url}/me",
+          'Connection' => 'keep-alive'
+          }
+      }
+    )        
+
+=begin
```
Comment (kcampos) [inline lib/ks/acal/acal.rb:147]: Add some commenting around this to explain what you're doing here

File: lib/ks/acal/acal.rb [hunk 6]
```diff
@@ -0,0 +1,183 @@
...
+  
+  def read_file(file)
+    contents = File.read(file)
+    boundary = contents.split(/\n/).first
+    
+    [boundary, contents]
+    
+  end  
+  
+  def scratch
```
Comment (kcampos) [inline lib/ks/acal/acal.rb:177]: Remove this empty method

File: lib/ks/utility/authentication.rb [hunk 7]
```diff
@@ -106,6 +106,43 @@ def login(opts={})
...
+  def acal_login(opts={})
+    defaults = {
+      :user => 'admin',
+      :password => 'admin',
+      :thinktime => 3
+    }
+    
+    opts = defaults.merge(opts)
+    
+#    @request.add('/')
```
Comment (kcampos) [inline lib/ks/utility/authentication.rb:118]: remove

**Comment by kcampos** [comment]
Resolve all previous in line comments and add commit to this pull request

File: lib/ks/acal/acal.rb (hunk 1 above)
Comment (kgirvan) [inline lib/ks/acal/acal.rb:67]: I have made all of the suggested corrections, but I am still working on the
request for the search. I will do a git push on the same git pull request.

On Mon, May 14, 2012 at 4:04 PM, Kyle Campos <
reply@reply.github.com

**Comment by kcampos** [comment]
new PR will address issues

**Pull request closed without merging**